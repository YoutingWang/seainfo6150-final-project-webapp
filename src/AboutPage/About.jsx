import React from "react";
import styles from "./About.module.css";
import about from "../images/about.jpeg";
import aboutme from "../images/AboutMe.JPG";
import Navigation from "../NavigationBar/Navigation";
const About = () => {
    return (
        <div>
            <Navigation/>
            <header className={styles.header}>Welcome</header>
            <img className={styles.background} src={about}
                alt="About"/>
            <p className={styles.text}>Here are some yummy restaurants, 
               we provide all kinds of restaurants in the world. Enjoy!</p>
            <h2 className={styles.header}>About me</h2>
            <img className={styles.img} src={aboutme} alt="about me"/>
            <p className={styles.text}>Youting Wang</p>
            <p className={styles.text}>Northeastern University</p>
            <p className={styles.text}>Information Systems</p>
            <p className={styles.text}>Enroll in 19 Fall</p>
            <p className={styles.text}>I am very interested in the food in the world. I also like to study how to make them. 
            I hope I can bring you my passion for food. Hope you can enjoy the food.</p>
        </div>
    );
};

export default About;