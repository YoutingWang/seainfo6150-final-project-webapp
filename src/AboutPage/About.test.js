import React from "react";
import {
    render
} from "@testing-library/react";
import About from "./About.jsx";

it("renders the home component correctly", () => {
    const {
        container
    } = render( < About / > );
    expect(container).toMatchSnapshot();
});