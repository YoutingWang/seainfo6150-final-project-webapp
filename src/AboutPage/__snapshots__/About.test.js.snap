// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <div>
    <div
      class="header"
    >
      <div
        class="left"
      >
        <img
          alt="Logo"
          class="logo"
          src="Logo.jpg"
        />
        <h1
          class="title"
        >
          Yummy Restaurants
        </h1>
      </div>
      <div
        class="right"
      >
        <button
          type="button"
        >
          Log in
        </button>
        <button
          type="button"
        >
          Sign up
        </button>
      </div>
    </div>
    <header
      class="header"
    >
      Welcome
    </header>
    <img
      alt="About"
      class="background"
      src="about.jpeg"
    />
    <p
      class="text"
    >
      Here are some yummy restaurants, we provide all kinds of restaurants in the world. Enjoy!
    </p>
    <h2
      class="header"
    >
      About me
    </h2>
    <img
      alt="about me"
      class="img"
      src="AboutMe.JPG"
    />
    <p
      class="text"
    >
      Youting Wang
    </p>
    <p
      class="text"
    >
      Northeastern University
    </p>
    <p
      class="text"
    >
      Information Systems
    </p>
    <p
      class="text"
    >
      Enroll in 19 Fall
    </p>
    <p
      class="text"
    >
      I am very interested in the food in the world. I also like to study how to make them. I hope I can bring you my passion for food. Hope you can enjoy the food.
    </p>
  </div>
</div>
`;
